handlers/meshery_application_handler.go

```diff
@@ -30,6 +30,20 @@ func (h *Handler) ApplicationFileHandler(
 	h.PatternFileHandler(rw, r, prefObj, user, provider)
 }
 
+// swagger:route POST /api/experimental/application/ ApplicationsAPI idPostApplicationFileRequest
+// Handle POST request for Application Files
+
+// Sends requests for Application files
+// responses:
+// 	200: applicationFileRequestWrapper
+
+// swagger:route GET /api/experimental/application/ ApplicationsAPI idGetApplicationFileRequest
+// Handle GET request for Application Files
+
+// Fetches requests for Application files
+// responses:
+// 	200: noContentWrapper
```

Comment:
you sure? with GET we are getting no response?

Reply:
I'm not a 100% sure, could you tell me if it is getting any response?

Reply:
Go through the handler maybe. That's where you'll find what's the response from the api.